src/components/Messages/MessageButtonGroup.jsx

```diff
@@ -39,7 +39,7 @@ const MessageButtonGroup = ({ showModal, setShowModal, boxType, setBoxType }) =>
         onClick={() => setShowModal(!showModal)}
         startIcon={<CreateIcon />}
         color="secondary"
-        sx={{ width: isReallySmallScreen ? '200px' : 'default' }}
+        sx={{ borderRadius: '25px' }}
```

Comment:
Shouldn't this be 20 as well?

Reply:
I started it at 20, and it didn't look correct. test it out. I feel the extra 5px did the trick.

Reply:
Had just took a look between 20px and 25px for the smaller buttons.

Visually, I think we could have the smaller buttons use 25px as well, there's not much of a difference. We could get that `border-radius` standardized for all our buttons.

Reply:
> Had just took a look between 20px and 25px for the smaller buttons.
> 
> Visually, I think we could have the smaller buttons use 25px as well, there's not much of a difference. We could get that `border-radius` standardized for all our buttons.

Fair enough, switched all to 25px...

---

Could you verify for me that nothing is being used in `MessageStyles.js` except for the `Pagination Container` styles?  We could really clean up that file if so.
`src > components > Messages > MessageStyles.js`

Reply:
> Could you verify for me that nothing is being used in `MessageStyles.js` except for the `Pagination Container` styles?  We could really clean up that file if so.
> `src > components > Messages > MessageStyles.js`

I need to get a more thorough check later, but it does look like we moved over most of the styles to MUI minus the pagination container.

Think a clean up would be good.

Reply:
> > Could you verify for me that nothing is being used in `MessageStyles.js` except for the `Pagination Container` styles?  We could really clean up that file if so.
> > `src > components > Messages > MessageStyles.js`
> 
> I need to get a more thorough check later, but it does look like we moved over most of the styles to MUI minus the pagination container.
> 
> Think a clean up would be good.

Feel like this should be done here in this PR?

Reply:
Think we could do that. Also think we could have it on a fresh PR to keep this separate. https://github.com/codeforpdx/PASS/pull/540#discussion_r1405295269

Reply:
Cleared unused code.